cmd/otelcontribcol/receivers_test.go

```diff
@@ -72,6 +72,10 @@ func TestDefaultReceivers(t *testing.T) {
 		{
 			receiver: "apache",
 		},
+		{
+			receiver: "apachespark",
+			skipLifecyle: true, // TODO: skipped until the receiver has is out of development status
```

Comment:
Can you help me understand why this is necessary? I don't recall ever seeing this as a reason for skipping a component here.

Reply:
You can temporarily embed `component.StartFunc` and `component.ShutdownFunc` into the empty receiver struct. ([example](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1d124cfa276e53b61bd303e6bbaea08c338592bb/connector/countconnector/connector.go#L41-L42)) I'd rather we do it this way because it ensures we can't forget to validate the functionality later.